src/main/java/mekanism/common/tile/TileEntityLogisticalSorter.java

```diff
@@ -46,32 +46,37 @@
 import net.minecraft.tileentity.TileEntity;
 import net.minecraft.util.EnumFacing;
 import net.minecraft.util.NonNullList;
-import net.minecraft.util.SoundCategory;
 import net.minecraftforge.common.capabilities.Capability;
 import net.minecraftforge.common.util.Constants.NBT;
 import net.minecraftforge.fml.common.FMLCommonHandler;
 import net.minecraftforge.items.CapabilityItemHandler;
 
-public class TileEntityLogisticalSorter extends TileEntityElectricBlock implements IRedstoneControl, IActiveState,
-      ISpecialConfigData, ISustainedData, ISecurityTile, IComputerIntegration {
+public class TileEntityLogisticalSorter extends TileEntityEffectsBlock implements IRedstoneControl,
+      ISpecialConfigData, ISustainedData, ISecurityTile, IComputerIntegration, IUpgradeTile {
 
     public HashList<TransporterFilter> filters = new HashList<>();
     public RedstoneControl controlType = RedstoneControl.DISABLED;
     public EnumColor color;
     public boolean autoEject;
     public boolean roundRobin;
+    public boolean singleItem;
     public int rrIndex = 0;
     public int delayTicks;
-    public boolean isActive;
     public boolean clientActive;
```

Comment:
I haven't tested this stuff yet, but my suspicion is that some stuff between clientActive/isActive is still duplicated now/not needed due to the changing superclass.

Reply:
Probably right.

Not sure how to test this, as I understand only the surface of this networking stuff, but I removed the `isActive` and `clientActive` logic from `TileEntityLogisticalSorter` and all seems good.

Reply:
Added back one small part you removed that made the sorter update so the light on the side would turn on. Given it turns out the `TileEntityEffectsBlock` doesn't call it quite often enough in specific cases at least with the sorter which should have it happen more often.